### PR TITLE
Support some Order APIs for Binance Futures

### PR DESCRIPTION
**Pull request by trexnix**
Support some APIs:

- [Create new orders](https://binanceapitest.github.io/Binance-Futures-API-doc/trade_and_account/#new-order-trade)

- [Query an order](https://binanceapitest.github.io/Binance-Futures-API-doc/trade_and_account/#query-order-user_data)

- [Cancel an order](https://binanceapitest.github.io/Binance-Futures-API-doc/trade_and_account/#cancel-order-trade)

- [Get current open orders](https://binanceapitest.github.io/Binance-Futures-API-doc/trade_and_account/#current-open-orders-user_data)

### CODE REVIEW

File: lib/binance/futures.ex [hunk 1]
```diff
@@ -147,4 +147,235 @@ defmodule Binance.Futures do
...
+
+    case HTTPClient.get_binance("/fapi/v1/openOrders", %{}, secret_key, api_key) do
+      {:ok, data} -> {:ok, Enum.map(data, &Binance.Futures.Order.new(&1))}
+      err -> err
+    end
+  end
+
+  def get_open_orders(symbol) when is_binary(symbol) do
+    api_key = Application.get_env(:binance, :api_key)
+    secret_key = Application.get_env(:binance, :secret_key)
```
Comment (sirius248) [inline lib/binance/futures.ex]: @trexnix can you move these info `api_key = Application.get_env(:binance, :api_key)` into attribute variables ?

File: lib/binance/futures.ex [hunk 2]
```diff
@@ -147,4 +147,235 @@ defmodule Binance.Futures do
...
+  """
+  def get_order(
+        symbol,
+        order_id \\ nil,
+        orig_client_order_id \\ nil
+      )
+      when is_binary(symbol)
+      when is_integer(order_id) or is_binary(orig_client_order_id) do
+    api_key = Application.get_env(:binance, :api_key)
+    secret_key = Application.get_env(:binance, :secret_key)
```
Comment (sirius248) [inline lib/binance/futures.ex]: This info is being duplicated in different places

File: lib/binance/futures/order.ex [hunk 3]
```diff
@@ -0,0 +1,21 @@
+defmodule Binance.Futures.Order do
```
Comment (sirius248) [inline lib/binance/futures/order.ex:1]: Wondering why we have this module `Binance.Futures.Order`, why not use this `Binance.Order` ?

**Comment by sirius248** [comment]
The PR look ok. I got some feedback. Can you have a look?

File: lib/binance/futures.ex (hunk 1 above)
Comment (trexnix) [reply lib/binance/futures.ex]: I updated @kimquy

File: lib/binance/futures/order.ex (hunk 3 above)
Comment (trexnix) [reply lib/binance/futures/order.ex:1]: Because Spot and Futures returns different response @kimquy

**Pull request merged**